packages/cozy-sharing/src/SharingBanner/hooks/useShouldDisplay.js

```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from 'react'
```

Comment:
This is a brand new file (did not exist in cozy-drive). 

We authorize the banner display by naively looking at the window location. If the location gets a match in an array of valid locations, we just return `true`.